packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -383,11 +386,12 @@ async function getResourceManagementClientForArmDeployment(
 }
 
 async function compileBicepToJson(
+  bicepCommand: string,
   bicepOrchestrationFilePath: string,
   jsonFilePath: string
 ): Promise<void> {
   // TODO: ensure bicep cli is installed
```

Comment:
Remove this todo comment since the ensure bicep cli step is done. Thx

Reply:
Removed